code/game/gamemodes/malfunction/Malf_Modules.dm

```diff
@@ -77,13 +77,13 @@
 	var/disable_text = "<span class='danger'>Goodbye Cruel World!</span>" //Context clues!
 	var/datum/action/innate/ai/ranged/attached_action
 
-/obj/effect/proc_holder/ranged_ai/Destroy()
-	QDEL_NULL(attached_action)
-	return ..()
+// /obj/effect/proc_holder/ranged_ai/Destroy()

```

Comment:
I'm not sure if this is necessary. This proc kept giving me runtimes about multiple qdels on the same object. When I commented them out, the runtimes went away. All the action buttons function normally and still get removed when they run out of charges so I don't think this is affecting anything.

Reply:
Confirmed runtime also exists on the original TG version